src/rrt/Tree.hpp

```diff
@@ -236,8 +264,13 @@ class Tree {
      */
     Node<T>* grow() {
         //  extend towards goal, waypoint, or random state depending on the
-        //  biases
-        //  and a random number
+        //  biases and a random number
+        if (_nodes.size() == 1) {
+            flann::Matrix<double>* root = new flann::Matrix<double>(
+                (double*)&(rootNode()->state()), 1,
+                sizeof(rootNode()->state()) / sizeof((double)(0.0)));
```

Comment:
Is `state` guaranteed to be convertible to a double pointer? I don't think the old code made that assumption